mappings/net/minecraft/entity/passive/BeeEntity.mapping

```diff
@@ -31,6 +31,14 @@ CLASS net/minecraft/class_4466 net/minecraft/entity/passive/BeeEntity
 		COMMENT A bee will start moving to a flower once this time in ticks has passed from a pollination.
 	FIELD field_30286 POLLINATION_FAIL_TICKS I
 		COMMENT The duration in ticks when a bee's pollination is considered failed.
+	FIELD field_30288 MAX_POLLINATED_CROPS I
+	FIELD field_30289 STING_POISON_DURATION_NORMAL_DIFFICULTY I
+	FIELD field_30290 STING_POISON_DURATION_HARD_DIFFICULTY I
+	FIELD field_30291 MAX_REACH_DISTANCE I
+		COMMENT The maximum distance a bee will wander from the hive before getting lost.
+		COMMENT Also the maximum distance to a flower.
+	FIELD field_30293 MAX_DISTANCE_FROM_HIVE I
```

Comment:
these sound like max and min wander range

Reply:
I think the javadoc of 30291 could be changed to "The maximum distance to a position before it is considered too far" 
because of the explanation below.

30291 is used only in `#isTooFar`, which is used in `MoveToFlowerGoal` and `MoveToHiveGoal`
`MoveToFlowerGoal`: when the bee isn't navigating checks whether the `#flowerPos` is too far, if it is, `#flowerPos` is set to null, if it isn't, the bee navigates to the flower.
`MoveToHiveGoal`: when the bee isn't navigating checks whether the `#hivePos` is more than 16 blocks away (`field_30293`), if it is and it is too far (`#isTooFar`) sets the bee as lost, if it isn't too far navigates back to the hive

Reply:
so it's the scan range for flower and hive respectively. if so consider naming them to show they are similar